src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -153,13 +164,18 @@ protected function renderHook($hookName, array $parameters)
 
     /**
      * Generates a documentation link
+     *
+     * @param string      $section Legacy controller name
+     * @param bool|string $title   Help title
+     *
+     * @return string
      */
     protected function generateSidebarLink($section, $title = false)
     {
         $version = $this->get('prestashop.core.foundation.version')->getVersion();
         $legacyContext = $this->get('prestashop.adapter.legacy.context');
 
-        if (empty($title)) {
+        if (false === $title) {
```

Comment:
empty is better in case we pass an empty string, 0, or anything useless. Otherwise use a proper test like `is_string && != `'' :)

Reply:
if its not false, that means you passed something to it, so maybe its intentional to have empty string or anything else? :thinking:

Reply:
Why not. I just prefer control everything I can :P

Reply:
Agree with @sarjon, sounds risky to change the behavior on a refactoring contribution: your choice @PierreRambaud: what do we do?

Reply:
Agreed with @mickaelandrieu , maybe some old modules are passing `null` / `''`.